creator-node/src/fileManager.js

```diff
@@ -80,7 +80,19 @@ async function saveFileToIPFSFromFS(
     enableIPFSAdd
   )
 
-  // store file copy by multihash for future retrieval
+  const dstPath = await copyFileToFs(multihash, srcPath, logger)
```

Comment:
lets actually not call this here and have these as separate functions? we shouldnt couple writing to ipfs and writing to disk. we did this because previously ipfs was a prerequisite but it's no longer one